### PR TITLE
docs: Fix grafana dashboard template mistake(#1181)

### PR DESCRIPTION
**Pull request by nature1995**
fix issue: https://github.com/juicedata/juicefs/issues/1181

### CODE REVIEW

File: docs/en/grafana_template.json [hunk 1]
```diff
@@ -15,19 +15,19 @@
...
   "links": [],
   "panels": [
     {
       "aliasColors": {},
       "bars": false,
       "dashLength": 10,
       "dashes": false,
-      "datasource": "$Source",
+      "datasource": "$datasource",
+      "description": "平均使用空间",
```
Comment (davies) [inline docs/en/grafana_template.json:28]: Please use English

**Comment by davies** [comment]
@nature1995 Thanks for sending out the PR, we will try it and get back to you.

`node` is good for K8S clusters, maybe we should create different template for K8S cluster and others.

**Comment by nature1995** [comment]
> @nature1995 Thanks for sending out the PR, we will try it and get back to you.
> 
> `node` is good for K8S clusters, maybe we should create different template for K8S cluster and others.

I think we can add another file `grafana_template_k8s.json`, just replace all `instances` with `node`.

**Comment by nature1995** [comment]
> @nature1995 Thanks for sending out the PR, we will try it and get back to you.
> 
> `node` is good for K8S clusters, maybe we should create different template for K8S cluster and others.

**Comment by nature1995** [comment]
> @nature1995 Thanks for sending out the PR, we will try it and get back to you.
> 
> `node` is good for K8S clusters, maybe we should create different template for K8S cluster and others.

File: docs/en/grafana_template.json (hunk 1 above)
Comment (nature1995) [reply docs/en/grafana_template.json]: I have been deleted the chinese part, add the k8s copy as well

**Comment by chnliyong** [comment]
We import the template on `Grafana Version 8.3.0 (commit: 914fcedb72, branch: HEAD)` and it doesn't work. 
image
If we change the **Min step** from `$interval` you added to a specified value like `1m`, it works. 
The doc [here](https://grafana.com/docs/grafana/latest/datasources/prometheus/#query-editor-in-dashboards) seems not support user defined variable:
> Use ‘Minimum’ or ‘Maximum’ step mode to set the lower or upper bounds respectively on the interval between data points. For example, set “minimum 1h” to hint that measurements are not frequent (taken hourly). Use the ‘Exact’ step mode to set a precise interval between data points. $__interval and $__rate_interval are supported.

**Comment by nature1995** [comment]
> We import the template on `Grafana Version 8.3.0 (commit: 914fcedb72, branch: HEAD)` and it doesn't work. image If we change the **Min step** from `$interval` you added to a specified value like `1m`, it works. The doc [here](https://grafana.com/docs/grafana/latest/datasources/prometheus/#query-editor-in-dashboards) seems not support user defined variable:
> 
> > Use ‘Minimum’ or ‘Maximum’ step mode to set the lower or upper bounds respectively on the interval between data points. For example, set “minimum 1h” to hint that measurements are not frequent (taken hourly). Use the ‘Exact’ step mode to set a precise interval between data points. $__interval and $__rate_interval are supported.

Do you select the sampling optioin which on the top, I think you need choice one from the list ? 
I have removed the interval option, please check the newest commit. If you have the same issue, please let me know.

**Pull request merged**